experimental/python/wheel.bzl

```diff
@@ -209,6 +212,11 @@ _entrypoint_attrs = {
     "console_scripts": attr.string_dict(
         doc = """\
 console_script entry points, e.g. 'experimental.examples.wheel.main:main'.
+""",
+    ),
+    "plugins": attr.string_dict(
```

Comment:
That should be string_list_dict to allow more than one entry point in single group, right?

Reply:
Yep.